tests/foreman/api/test_docker.py

```diff
@@ -695,11 +728,37 @@ def test_publish_multiple_docker_repo_composite_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        product = entities.Product(organization=self.org).create()
+        repo = _create_repository(product)
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.org,
+        ).create()
+        content_view.repository = [repo]
+        content_view = content_view.update(['repository'])
+        self.assertIn(repo.id, [repo_.id for repo_ in content_view.repository])
+        self.assertIsNone(content_view.read_json()['last_published'])
+
+        content_view.publish()
+        self.assertIsNotNone(content_view.read_json()['last_published'])
+        cvv = content_view.read().version[0].read()
+
+        comp_content_view = entities.ContentView(
+            composite=True,
+            organization=self.org,
+        ).create()
+        comp_content_view.component = [cvv]
+        comp_content_view = comp_content_view.update(['component'])
+        self.assertEqual(cvv.id, comp_content_view.component[0].id)
```

Comment:
Gah. Again, I goobered up. I should have used a list comprehension, as I did elsewhere.

```python
self.assertEqual([cvv.id], [comp.id for comp in comp_content_view.component])
```